stack/base/before-notebook.d/10_prepare-home-rc.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# If the container is start by spawner and the home is remounted.
```

Comment:
Can you explain a bit more how is this home remounting different from the mounting that we do normally e.g. when using `aiidalab-launch`? 
Does this mean that there's a pre-existing `.bashrc` in the mounted volume that overrides the one from the image? Or something else?

Reply:
I have no exact clue why it is overrided. I suspect the order of creating persistence volume after the container is created matters. I'll try to do more investigation see if there any document mentioned the order of mounting order. I need to compare with aiidalab-launch why there is no issue there since it creates volume and mount but not these issues.

Reply:
Could it be due to the home mount configuration for the Kubernetes cluster?